File: WebApi/CodeGenerator.DOMAIN/Models/ApplicationDto.cs
using CodeGenerator.DOMAIN.Models.Db;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CodeGenerator.DOMAIN.Models
{
    public class ApplicationDto
    {
        public string ApplicationName { get; set; }
        public string Path { get; set; }
        public string TemplateName { get; set; }
        public DbInformation DbInformation { get; set; }
    }
}

File: UI/CodeGenerator/src/app/domain/application-dto/application-dto.ts
import { DbInformation } from "../db-information/db-information";

export class ApplicationDto {
    ApplicationName : string = "";
    Path : string = "";
    TemplateName : string = "";
    VariableTypes : string = "";
    Language : string = "";
    DbInformation : DbInformation = new DbInformation();
}
